<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManager</class>
 <widget class="QWidget" name="UserManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ユーザー管理システム</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_userList">
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>名前</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>年齢</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_userName"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_searchByUserName">
       <property name="text">
        <string>名前で検索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_getAllUsers">
       <property name="text">
        <string>すべて取得</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_newUserName"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_age"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addUser">
       <property name="text">
        <string>登録</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
